sources/bot/modules/spreadsheet/study_staff/study_staff_spreadsheet_handler.py

```diff
@@ -0,0 +1,41 @@
+from typing import List, Dict
+
+from bot.modules.spreadsheet.spreadsheet_handler import SpreadsheetHandler
+
+
+class StudyStaffSpreadsheetHandler:
```

Comment:
I think we need to implement an interface for this class

Reply:
done